### DISCUSSION TITLE
pdf-Grösse gewachsen, sinnvoll?

### DISCUSSION THREAD

**Discussion by mge-medisoftware** [Digitaler-Impfnachweis/certification-apis#68] [Q&A]
Mhhhm, die pdf-Grösse ist vermutlich wegen der Anleitung zum Falten von 29 kB auf 198 kB gewachsen.
Nun ist die Internetanbindung einiger Praxen "suboptimal". Ob die zusätzliche Info evtl. resultierende Probleme Wert ist?

**Comment by dirkexe** [comment]
Die Anleitung zum Falten scheint sauber aus Vektorobjekten im PDF zu bestehen, die keinen Platz verschwenden. Der QR-Code jedoch ist jetzt mit 24 Bit Farbtiefe im PDF kodiert. Dabei würde 1 Bit / monochrom ausreichend sein. Hier besteht aus meiner Sicht auch Optimierungsbedarf und -potential.

**Reply by mge-medisoftware** [reply]
Oh ja, stimmt. Interessanterweise haben die Kästchen im QR-Code jetzt graue Ränder als ob Kantenglättung durchgeführt würde, was normalerweise ja bei senkrechten oder waagerechten Elementen eher nicht nötig ist. Ob das die Erkennungsrate beim Scannen verbessert?

**Reply by thinkberg** [reply]
Checken wir, das soll natuerlich nicht sein.

**Reply by thinkberg** [reply]
oder bezieht sich das auf das template?

**Reply by dirkexe** [reply]
Wir beziehen uns auf das PDF, welches die API (`/api/certify/v2/issue`) als `application/pdf` liefert. Also jedes einzelne Zertifikat als PDF.

**Reply by thinkberg** [reply]
Wir haben das nochmal angesehen und es sollte wieder kleiner sein.

**Reply by mge-medisoftware** [reply]
Nun ca. 35kB. Prima!
